integration_tests/run_pyspark_from_build.sh

```diff
@@ -99,41 +99,29 @@ else
     ## Under cloud environment, overwrite the '--std_input_path' param to point to the distributed file path
     INPUT_PATH=${INPUT_PATH:-"$SCRIPTPATH"}
 
+    RUN_TESTS_COMMAND="$SCRIPTPATH"/runtests.py\ --rootdir\ "$LOCAL_ROOTDIR"\ "$LOCAL_ROOTDIR"/src/main/python
```

Comment:
I think this is equivalent and simpler to read.
```suggestion
    RUN_TESTS_COMMAND="$SCRIPTPATH/runtests.py --rootdir $LOCAL_ROOTDIR $LOCAL_ROOTDIR/src/main/python"
```

However with this PR's refactoring there is another layer of shell parsing.  I believe that causes issues when any of the path variables expand to paths with spaces in them (or other shell-sensitive characters when re-parsed).

Reply:
yes, it's quite likely it does not for edge cases. I presumed it does not apply to our test infra. If necessary I will look for a solution, and open for suggestions. 

re: suggested removal of the double-quotes I agree, I decided to keep it from existing code because it's not breaking anything afaik.

Reply:
>  I presumed it does not apply to our test infra.

This script is not specific to our infra.  It can run in any environment with the requisite Python modules and has `SPARK_HOME` setup.  Any user pulling the repo into a parent directory with a space in it will not be able to run this script after this change but could before it.